packages/openactive-broker-microservice/app.js

```diff
@@ -190,18 +193,37 @@ function withOpportunityRpdeHeaders(getHeadersFn) {
 
 /**
  * @param {string} baseUrl
- * @param {string} feedIdentifier
+ * @param {string} feedContextIdentifier
```

Comment:
Trying to improve consistency. So that `feedContextIdentifier` always refers to the string which can be `OrdersFeed (auth:secondary)`. `feedIdentifier` is used separately to refer to just the `OrdersFeed` part